### ISSUE TITLE
ReinerSCT cyberJack Komfort/Standard: SCARD_E_NOT_TRANSACTED

### ISSUE THREAD

**Issue by misery** [misery/DockerAusweisApp2#1]
Authentication with a cyberJack komfort/standard fails with "SCARD_E_NOT_TRANSACTED" after user enters PIN.

```
Establish connection using PIN
Starting PACE for PACE_PIN
SCardControl cmdBuffer: "02040303157f4c12.....1bda"
SCardControl for "REINER SCT cyberJack RFID komfort (9721581331) 00 00" : "SCARD_E_NOT_TRANSACTED" ""
Control to establish PACE channel failed
Finished PACE for PACE_PIN with result COMMAND_FAILED
```

This looks like a pcsc driver bug.

**Comment by misery** [comment]
Added bug to upstream: https://github.com/LudovicRousseau/PCSC/issues/22

**Comment by misery** [comment]
Fixed by eab3615dd018a075e02ea6bb768478e61b9aa394.
